client/navigation/Routes.tsx

```diff
@@ -12,29 +12,36 @@ export default function Routes() {
     const { user, setUser, userToken, setUserToken, userProfile, setUserProfile } = useContext(AuthContext);
     const [loading, setLoading] = useState(true);
     const [initializing, setInitializing] = useState(true);
+    const [enter, setEnter] = useState(false);
     
     // handle user state changes
     const onAuthStateChanged = async (user: any) => {
+        setEnter(false);
```

Comment:
What does this do?

Reply:
I think 'enter' is a little confusing. maybe isAuthenticated? or isAuthorized? or both since we're dealing with authentication and locked accounts.